eng/pipelines/templates/stages/platform-matrix-cryptography-dependency.json

```diff
@@ -28,13 +28,6 @@
       },
       "PythonVersion": "3.7",
       "CoverageArg": "--disablecov"
-    },
```

Comment:
This entirely eliminates windows. You can remove one of the versions from above, and update the current target to whatever you choose.